src/server/model/parser.py

```diff
@@ -0,0 +1,291 @@
+import re
+import copy
+import json
+import graphviz
+import utils as utils
+from enum import Enum
+from redbaron import RedBaron
+from multiprocessing import Process
+from model.modelGraph import ModelGraph
+
+pnls_utils = utils.PNLUtils()
+
+
+class PNLTypes(Enum):
+    COMPOSITIONS = 'Composition'
+    MECHANISMS = 'Mechanism'
+    PROJECTIONS = 'Projection'
+    SUMMARY = 'Summary'
+
+
+class ModelParser:
```

Comment:
This class looks quite complex, I would suggest to create a new technical debt card to split it into different smaller classes for the different responsibilities (f.e. ASTParsing and APIHandling)

Reply:
Agree and I was already aware, unfortunately I had to speed things up a bit for a demo with the client, I had the same idea to open a tech debt card for the future. Thx